modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -40,11 +40,11 @@
 
 | *Deployment* 
 | 
-    * Deploy using ROSA CLI
+    * Deploy using the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
The first instance of expanding ROSA should probably be in the first paragraph of this module, rather than here - I'd suggest updating the intro paragraph and just use "ROSA CLI" here.

Reply:
Updated the intro paragraph. Maybe "the ROSA CLI (`rosa`)" here?